### PR TITLE
updating link in the microsoftdocs legal notices

### PR DESCRIPTION
**Pull request by jamesiri**

### CODE REVIEW

**Comment by jeffwilcox** [comment]
I do not recommend taking this change, we should point to fwlinks or aka.ms links and never directly to a page for this kind of broad asset.

File: projections/microsoft.docs/README.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ see the [LICENSE](LICENSE) file, and grant you a license to any code in the repo
 Microsoft, Windows, Microsoft Azure and/or other Microsoft products and services referenced in the documentation
 may be either trademarks or registered trademarks of Microsoft in the United States and/or other countries.
 The licenses for this project do not grant you rights to use any Microsoft names, logos, or trademarks.
-Microsoft's general trademark guidelines can be found at [Microsoft Trademark and Brand Guidelines](http://go.microsoft.com/fwlink/?LinkID=254653).
+Microsoft's general trademark guidelines can be found at [Microsoft Trademark and Brand Guidelines](https://www.microsoft.com/en-us/legal/intellectualproperty/trademarks).
```
Comment (12Kirk2v121BTMST) [inline projections/microsoft.docs/README.md:26]: LICENSE

File: projections/microsoft.docs/README.md [hunk 2]
```diff
@@ -23,7 +23,7 @@ see the [LICENSE](LICENSE) file, and grant you a license to any code in the repo
...
 
 Privacy information can be found at [https://privacy.microsoft.com](https://privacy.microsoft.com).
```
Comment (12Kirk2v121BTMST) [inline projections/microsoft.docs/README.md:28]: https://api.govinfo.gov/published/{dateIssuedStartDate}/{dateIssuedEndDate}### 
```suggestion
Privacy information can be found at [https://privacy.microsoft.com](https://privacy.microsoft.com).
```

**Comment by 12Kirk2v121BTMST** [comment]
msDocsLinkFix

**Pull request closed without merging**